src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -506,6 +480,7 @@ AmendmentTableImpl::persistVote(
     bool vote_to_veto) const
 {
     auto db = db_.checkoutDb();
+    voteAmendment(*db, amendment, name, vote_to_veto);
     soci::transaction tr(*db);
     std::string sql =
```

Comment:
You added the function call, but didn't remove the raw SQL

Reply:
Done.